Configuration/StandardSequences/python/Reconstruction_cff.py

```diff
@@ -50,16 +50,16 @@
 localreco = cms.Sequence(bunchSpacingProducer+trackerlocalreco+muonlocalreco+calolocalreco+castorreco)
 localreco_HcalNZS = cms.Sequence(bunchSpacingProducer+trackerlocalreco+muonlocalreco+calolocalrecoNZS+castorreco)
 
-_phase2_localreco = localreco.copyAndExclude([castorreco])
-_phase2_localreco_HcalNZS = localreco_HcalNZS.copyAndExclude([castorreco])
-from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
-phase2_common.toReplaceWith(localreco, _phase2_localreco)
-phase2_common.toReplaceWith(localreco_HcalNZS, _phase2_localreco_HcalNZS)
+_run3_localreco = localreco.copyAndExclude([castorreco])
```

Comment:
@hvanhaev 
please confirm that there will be no running with castor after LS2.
Thank you.